solanum-tuberosums/djangazon
# Br orderdetails

## Associated Ticket
[UserCanCompleteOrder](https://github.com/solanum-tuberosums/djangazon/issues/2)
[UserCanDeleteFromShoppingCart](https://github.com/solanum-tuberosums/djangazon/issues/6)

## Description 
This should pull the order for the user and all of the products on the order or display no active order if there are no orders.

#### To Test:
You will need to delete your DB and migrations
makemigrations
migrate 
builddb

You should be able to manual type order/# into your url to test pulling specific orders. (security problems for later?)

## All Tests Pass
